MSVC syntax error

```
D:\chainer\chainerx_cc\chainerx\backward_builder.cc(162,108): error C4576:  a parenthesized type followed by an initializer list is a non-standard explicit type conversion syntax [D:\chainer\build\temp.win-amd64-3.7\Release\chainerx\chain erx.vcxproj]
```
https://github.com/chainer/chainer/blob/b1128b39805bba8e5af3e90eb547c4f5d6d88c51/chainerx_cc/chainerx/backward_builder.cc#L157-L165

With latest Visual Studio, line 162 caused a syntax error

--- a/chainerx_cc/chainerx/backward_builder.cc
+++ b/chainerx_cc/chainerx/backward_builder.cc
@@ -159,7 +159,7 @@ std::vector<RetainedInputToken> BackwardBuilder::RetainInput(std::vector<size_t>
     for (size_t i : indices) {
         CHAINERX_ASSERT(i < inputs_.size());
         input_retention_record_.Record(i);
-        token.emplace_back(RetainedInputToken(internal::GetArrayBody(gsl::at(inputs_, i))->GetParams(), i));
+        token.emplace_back(internal::GetArrayBody(gsl::at(inputs_, i))->GetParams(), i);
     }
     return token;
 }
@@ -175,7 +175,7 @@ std::vector<RetainedOutputToken> BackwardBuilder::RetainOutput(std::vector<size_
     for (size_t i : indices) {
         CHAINERX_ASSERT(i < outputs_.size());
         output_retention_record_.Record(i);
-        token.emplace_back(RetainedOutputToken(internal::GetArrayBody(gsl::at(outputs_, i))->GetParams(), i));
+        token.emplace_back(internal::GetArrayBody(gsl::at(outputs_, i))->GetParams(), i);
     }
     return token;
 }

--- a/chainerx_cc/chainerx/backward_builder.cc
+++ b/chainerx_cc/chainerx/backward_builder.cc
@@ -159,7 +159,7 @@ std::vector<RetainedInputToken> BackwardBuilder::RetainInput(std::vector<size_t>
     for (size_t i : indices) {
         CHAINERX_ASSERT(i < inputs_.size());
         input_retention_record_.Record(i);
-        token.emplace_back((RetainedInputToken){internal::GetArrayBody(gsl::at(inputs_, i))->GetParams(), i});
+        token.emplace_back(RetainedInputToken(internal::GetArrayBody(gsl::at(inputs_, i))->GetParams(), i));
     }
     return token;
 }
@@ -175,7 +175,7 @@ std::vector<RetainedOutputToken> BackwardBuilder::RetainOutput(std::vector<size_
     for (size_t i : indices) {
         CHAINERX_ASSERT(i < outputs_.size());
         output_retention_record_.Record(i);
-        token.emplace_back((RetainedOutputToken){internal::GetArrayBody(gsl::at(outputs_, i))->GetParams(), i});
+        token.emplace_back(RetainedOutputToken(internal::GetArrayBody(gsl::at(outputs_, i))->GetParams(), i));
     }
     return token;
 }

